### PR TITLE
Feat:  failed exports

### PR DESCRIPTION
**Pull request by SamerIssam**
## Description

This ticket shows the  number of products that failed to export.

## Testing instructions

Try to export products, if there is any product(s) failed it will show how many has failed.

### CODE REVIEW

File: .github/workflows/fdbt-site_test_deployment.yml [hunk 1]
```diff
@@ -127,6 +127,7 @@ jobs:
     - name: Sleep for 10 mins
       run: sleep 10m
       shell: bash
+
```
Comment (alainajaffer) [inline .github/workflows/fdbt-site_test_deployment.yml:130]: unnecessary line change

Comment (SamerIssam) [reply .github/workflows/fdbt-site_test_deployment.yml:130]: it seems this is the default ?

File: repos/exporter/lib/handler.ts [hunk 2]
```diff
@@ -135,6 +140,24 @@ export const handler: Handler<ExportLambdaBody> = async ({ paths, noc, exportPre
         }),
     );
 
+    const date = new Date();
+    const numberOfExpectedNetexFiles = paths.length;
+    const metadata = { date, numberOfExpectedNetexFiles };
+
+    console.log(
+        `putting metadata: date - ${metadata.date.toDateString()} numberOfExpectedNetexFiles - ${
+            metadata.numberOfExpectedNetexFiles
```
Comment (alainajaffer) [inline repos/exporter/lib/handler.ts:149]: Do we still want this here?

Comment (SamerIssam) [reply repos/exporter/lib/handler.ts:149]: @RobCattonKPMG  it can be good for logging to have it ?

**Comment by alainajaffer** [comment]
Screenshot 2022-11-10 at 11 04 13

**Review by alainajaffer — DISMISSED** [review]

**Review by alainajaffer — APPROVED** [review]

**Pull request merged**